diff --git a/SplitSource.php b/SplitSource.php
index da25789..5563bfc 100644
--- a/SplitSource.php
+++ b/SplitSource.php
@@ -381,7 +381,7 @@ function split_file($file, $chunks, $undo) {
             'language_pack_manager[_(-]|LanguagePackManager' => 'LanguagePackManager',
             'link_manager[_(-](?![.]get[(][)])|LinkManager' => 'LinkManager',
             'LogeventIdWithGeneration|add_log_event|delete_log_event|get_erase_log_event_promise|parse_time|store_time' => 'logevent/LogEventHelper',
-            'MessageContentUploadId' => 'MessageContentUploadId'
+            'MessageContentUploadId' => 'MessageContentUploadId',
             'MessageCopyOptions' => 'MessageCopyOptions',
             'MessageCover' => 'MessageCover',
             'MessageEffectId' => 'MessageEffectId',
diff --git a/example/java/org/drinkless/tdlib/example/Example.java b/example/java/org/drinkless/tdlib/example/Example.java
index 5218e5f..cbba453 100644
--- a/example/java/org/drinkless/tdlib/example/Example.java
+++ b/example/java/org/drinkless/tdlib/example/Example.java
@@ -293,7 +293,7 @@ public final class Example {
             new TdApi.InlineKeyboardButton("https://telegram.org?1", 0, null, new TdApi.InlineKeyboardButtonTypeUrl()),
             new TdApi.InlineKeyboardButton("https://telegram.org?2", 0, null, new TdApi.InlineKeyboardButtonTypeUrl()),
             new TdApi.InlineKeyboardButton("https://telegram.org?3", 0, null, new TdApi.InlineKeyboardButtonTypeUrl())};
-        TdApi.ReplyMarkup replyMarkup = new TdApi.ReplyMarkupInlineKeyboard(new TdApi.InlineKeyboardButton[][]{row, row, row});
+        TdApi.ReplyMarkup replyMarkup = new TdApi.ReplyMarkupInlineKeyboard(new TdApi.InlineKeyboardButton[][]{row, row, row}, false);
 
         TdApi.InputMessageContent content = new TdApi.InputMessageText(new TdApi.FormattedText(message, null), null, true);
         client.send(new TdApi.SendMessage(chatId, null, null, null, replyMarkup, content), defaultHandler);
